factom_sdk/client/anchors_client.py

```diff
@@ -19,15 +19,9 @@ def get(self, **kwargs):
         base_url = kwargs.get("base_url")
         app_id = kwargs.get("app_id")
         app_key = kwargs.get("app_key")
-        object_hash = kwargs.get("object_hash")
-        height = kwargs.get("height")
-        if object_hash is None:
-            assert height is not None, "either object_hash or height must not be None"
-            assert isinstance(height, int) and height > 0, "height must be a positive integer"
-            return self.request_handler.get("/".join([factom_sdk.utils.consts.ANCHORS_URL, str(height)]),
-                                            base_url=base_url, app_id=app_id, app_key=app_key)
-        else:
-            assert isinstance(object_hash, str) and len(object_hash) == 64, "object_hash must be a string of length 64"
-            assert height is None, "object_hash provided, height must be None"
-            return self.request_handler.get("/".join([factom_sdk.utils.consts.ANCHORS_URL, object_hash]),
-                                            base_url=base_url, app_id=app_id, app_key=app_key)
+        param = kwargs.get("object_identifier")
+        is_valid_param = (isinstance(param, int) and param >= 0) or (isinstance(param, str) and len(param) == 64)
+        assert is_valid_param, "object_identifier parameter must be a positive int or a string of length 64"

```

Comment:
while, yes, it needs to be a string of length 64 to pass your check, it seems to me the error message should be "...positive int or a valid sha256 hash"

Reply:
You can't tell that something is a "valid sha256 hash" unless you have the data that was hashed. You mean "must be lowercase hex string of length 64"?

Reply:
I'm fine with however you see best.

Reply:
Word, I don't think we check for hex digits in any other api client so I think length check is sufficient enough.